documents/tinyengine/extension-capabilities-tutorial/ai-plugin-configuration.md

```
@@ -0,0 +1,55 @@
+# AI 插件使用前配置
+
+## 一、AI 插件使用在本地代码里调试及要改动的配置，以文心一言举例
+
+### 后端代码改动
+
+1、在`tiny-engine-webservice/config/config.default.ts`文件变更如下
+
+![](./imgs/ai-image21.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)